src/quo2/components/wallet/account_card/view.cljs

```diff
@@ -6,15 +6,17 @@
             [quo2.components.buttons.button.view :as button]
             [quo2.components.markdown.text :as text]
             [utils.i18n :as i18n]
-            [quo2.theme :as theme]))
+            [quo2.theme :as theme]
+            [reagent.core :as reagent]
+            [quo2.foundations.customization-colors :as customization-colors]))
 
 (defn- loading-view
   [{:keys [customization-color type theme metrics?]}]
   (let [watch-only? (= :watch-only type)
         empty-type? (= :empty type)]
     [rn/view
      {:accessibility-label :loading
-      :style               (style/card customization-color watch-only? metrics? theme)}
+      :style               (style/card customization-color watch-only? metrics? theme false)}
```

Comment:
This function should take a map imo, once the props start to go beyond 3 a map is cleaner generally 👌

Reply:
Done